projects/crypto-currency/destemidos/Api/routes/singin.js

```diff
@@ -1,7 +1,26 @@
 import app from "./configs/app.js"
+import { Mongoose, UserSchema } from './configs/db.js'
 
-app.get('/login', (req,res)=>{
-    res.send('Fazer o login')
+app.post('/singin', async (req, res) => {
+ 
+  const email = req.body.email
+  const password = req.body.password
```

Comment:
Oque acha de fazer assim?
```suggestion
  const { email, password } = req.body
```

Reply:
Feito a desestruturação